distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/platform/TestSecurity.java

```diff
@@ -493,25 +492,6 @@ public void testBasicRestAccess() throws Exception {
         getSecurityPolicy().waitForGuestAuthReady(url);
     }
 
-    @Test(expected = NoHttpResponseException.class)
-    public void testBasicAuthOverHttp() throws Exception {
```

Comment:
❓ Why does this test not add value?  It looks like this is our test to make sure HTTP is disabled.

@codice/security